common/views/components/Contact/Contact.V2.tsx

```diff
@@ -34,11 +31,12 @@ const PhoneNumber = styled.span.attrs({ className: font('intr', 4) })`
   display: block;
 `;
 
-const WithIconWrapper = styled.div`
-  align-items: flex-start;
+const WithIconWrapper = styled(Space).attrs({
+  v: { size: 's', properties: ['margin-bottom'] },
+})`
   display: flex;
+  align-items: center;
   gap: 10px;
```

Comment:
not part of this PR, but just spotted this and while I remember, we are now in a place to update all instances of `gap: {x}px` with their responsive equivalent using the `Space` component.

Reply:
Ah yes good point, I'll make the change thanks!

Reply:
It does mean that the margins and the gaps have to have the same value, right? (aka can't have "small" for vertical margins  and "medium" for column gaps)?

I think it's fine in this case, it lessens the gap, but I'll suggest it to Dom.